index.bs

```diff
@@ -1798,6 +1822,66 @@ Note: Invoking this method from a [=browsing context=] where the [=Web Authentic
 
 </div>
 
+### Client Supported Capabilities (dictionary <dfn dictionary>SupportedCapabilities</dfn>) ### {#dictionary-supportedcapabilities}
+
+<xmp class="idl">
+    dictionary SupportedCapabilities {
+        boolean areU2FDevicesSupported;
+        boolean areCTAPDevicesSupported;
+        boolean areResidentKeysSupported;
+        boolean isUserVerificationSupported;
+        boolean isTokenBindingSupported;
+        UserVerifyingPlatformAuthenticatorAvailability userVerifyingPlatformAuthenticatorAvailability;
+        sequence<AuthenticatorTransport> transportsSupported;
+        sequence<DOMString> extensionsSupported;
+    };
+</xmp>
+
+This dictionary is used by [=[WRPS]=] to determine the supported capabilities of the [=client=].
+
+<div dfn-type="dict-member" dfn-for="SupportedCapabilities">
+    :   <dfn>areU2FDevicesSupported</dfn>
+    ::  This attribute indicates whether the [=client=] supports communicating with [[FIDO-U2F-Message-Formats]] devices.
+
+    :   <dfn>areCTAPDevicesSupported</dfn>
+    ::  This attribute indicates whether the [=client=] supports communicating with [[FIDO-CTAP]] devices.
+
+    :   <dfn>areResidentKeysSupported</dfn>
+    ::  This attribute indicates whether the [=client=] supports creating [=resident credentials=].
+
+    :   <dfn>isUserVerificationSupported</dfn>
+    ::  This attribute indicates whether the [=client=] supports [=user verification=].
+
+    :   <dfn>isTokenBindingSupported</dfn>
+    ::  This attribute indicates whether the [=client=] supports the [=Token Binding=] protocol.
+
+    :   <dfn>userVerifyingPlatformAuthenticatorAvailability</dfn>
+    ::  This attribute indicates the availability of a [=user-verifying platform authenticator=] on the [=client=].
+
+    :   <dfn>transportsSupported</dfn>
```

Comment:
We have to define the strict ordering of these two sequences to reduce fingerprinting opportunity.

Reply:
Would it make sense to define them as [ordered sets](https://infra.spec.whatwg.org/#sets)? I agree we should probably also explicitly define the order like we have for [`[[transports]]`](https://www.w3.org/TR/webauthn-2/#dom-authenticatorattestationresponse-transports-slot).

Reply:
For a list, I always default to saying that it should be ordered lexicographically and not contain duplicates. Looks like [ordered sets](https://infra.spec.whatwg.org/#sets) imply the uniqueness condition, so that would work.

Reply:
I have applied the same logic as was previously done for transports.